ios/TeamHome/TeamHome/Models/Mock json/MessageExample.json

```diff
@@ -0,0 +1,27 @@
+ {
```

Comment:
Property names in JSON files should be double quoted. This isn't properly formed JSON.